pytorch_lightning/accelerators/gpu.py

```diff
@@ -99,7 +96,7 @@ def name() -> str:
         return "gpu"
 
 
-def get_nvidia_gpu_stats(device: _DEVICE) -> Dict[str, float]:
+def get_nvidia_gpu_stats(device: _DEVICE) -> Dict[str, float]:  # pragma: no-cover
```

Comment:
Unused, but would need a deprecation

Reply:
this utility is a bit more convenient in terms of the logs that it generates. The one from PT creates a lot of logs that are hard to interpret. I saw a user asking to allow them to somehow differentiate between these 2, either on issues/slack somewhere.